### ISSUE TITLE
Parcel dependency is missing from before-using-agoric.md

### ISSUE THREAD

**Issue by bhgsbatista** [Agoric/documentation#277]
Parcel is required for to run the demo UI, but it's not listed as a dependency.

**Comment by katelynsills** [comment]
Hi Samuel, thanks for this report. Can you tell me more about what kind of experience you had, without parcel installed previously? Did you get an error? Did you have to run an additional `yarn install`?

**Comment by bhgsbatista** [comment]
No, I add to do `yarn add parcel -g` to make it available.  In the dapp-encouragement UI's package.json there are the following scripts:

```
"build": "parcel build public/index.html",
"start": "parcel serve --host=127.0.0.1 --port=3000 public/index.html"
```

in order to avoid having to install parcel globally, these should be:

"build": "./node_modules/parcel build public/index.html",

**Comment by katelynsills** [comment]
Ok, thanks so much! I'm going to make this an issue on the `dapp-encouragement` repo so we can get this fixed.
